examples/source/advertising-analytics/Tracking_Core_Web_Vitals.html

```diff
@@ -99,3 +99,48 @@
       }
     </script>
   </amp-analytics>
+
+
+  <!--
+  ## Comparing performance on Cache vs Origin
+  -->
+  <!--
+    AMP Caches can improve your code with a number of optimizations, but it is important to not rely on a Cache for your site's performance. A good way to make sure you are delivering a great experience for all your users is to keep track of how your pages' Core Web Vitals are performing on and off of the Cache.
+
+We can build on our Google Analytics to do this. One of substitutions provided by amp-analytics is `ampdocHost` - this lets us track the URL of the page the user visited. We exand our earlier config using `extraUrlParams`. From there, we need to [set up](https://support.google.com/analytics/answer/2709829) a new Custom Dimension to track this. Make sure to take note of the number of the Custom Dimension - below we assume it was "1", in `cd1`..
+  -->
+  <div>
+    <amp-analytics>
+      <script type="application/json">
+        {
+          "vars": {
+             "gtag_id": "UA-XXXXXXXXX-X",
+             "config": {
+               "UA-XXXXXXXXX-X": {
+                  "groups": "default"
+               }
+             }
+          },
+          "extraUrlParams": {
+              "cd1": "${ampdocHost}"
+          },
+          "requests": {
+            "event": "<% base_path %>/ping?user=[= user =]&account=ampdev&event=${eventId}",
```

Comment:
Is this a proper GA request?

Reply:
it is just copied from the section above this one, @micajuine-ho added it I believe

Reply:
Would be good to clarify.

Reply:
@micajuine-ho would you be able to clarify? I don't know the background on this one

Reply:
just realized this was a typo on my part, fixed by removing that line